IamCredentials/tests/Unit/V1/IAMCredentialsClientTest.php

```diff
@@ -20,18 +20,18 @@
  * This file was automatically generated - do not edit!
  */
 
-namespace Google\Cloud\Iam\Credentials\Tests\Unit\V1;
+namespace Google\Iam\Credentials\Tests\Unit\V1;
```

Comment:
This is a BC-Breaking change, and probably shouldn't have happened

Reply:
This was changed by the protos and we need to decide if PHP should preserve the previous (cloud) namespace or not.

Because this library is beta, we could potentially change it. But for consistency we'd then want to change the from `google-cloud-php-iam` to `iam-php`, change the composer package, etc.

Reply:
I think we would like to keep `Cloud` in the name, it is what other languages such as .NET and Java have done.